src/charactersheet/viewmodels/common/root/index.html

```diff
@@ -76,13 +73,10 @@
               >
                 <i class="fa fa-usd"></i>&nbsp;&nbsp;Support Us </a>
```

Comment:
Unrelated to this PR, but I wonder if the icon for `Support Us` might be better as the patreon logo. Maybe people are more inclined to click support if they see our patreon? idk

Reply:
I agree. The patriot logo wasn’t in font awesome in the version we’re using. I can definitely make another PR where I replace the logo